logs.yml

```
@@ -0,0 +1,98 @@
+---
+- name: Zookeeper Provisioning
+  hosts: zookeeper
+  gather_facts: false
+  tags: zookeeper
+  tasks:
+    - import_role:
+        name: confluent.variables
+    - include_tasks: 
+        file: roles/confluent.common/tasks/failure_handling.yml
+      vars:
+        service_name: "{{ zookeeper_service_name }}"
+        config_file: "{{ zookeeper.config_file }}"
+        log_dir: "{{zookeeper_log_dir}}"
```

Wondering if this could be simplified to:

```suggestion
    - include_role:
        name: confluent.common
        tasks_from: failure_handling.yml
      vars:
        service_name: "{{ zookeeper_service_name }}"
        config_file: "{{ zookeeper.config_file }}"
        log_dir: "{{zookeeper_log_dir}}"
```

Might not work bc you need the vars set... The reason im curious is the variables role is a dependency of the common role.

---

You're right, it works with variables as dependency.